spellcheck.sh

```diff
@@ -9,21 +9,46 @@
 #   1) removes all .diff files in "." directory and its sub-directories
 
 ignore="! -iname colors.tex"
+CLEAN=0
+
+# Flags handler
+while test $# -gt 0; do
+  case "$1" in
+    -h|--help)
+      echo "texspell-- Command line spell-checker tool for TeX documents"
+      echo " "
+      echo "Options:"
+      echo "-h, --help: get this help"
+      echo "-c, --clean: Remove all .diff in the . directiory and sub-directories"
+      echo "-v, --version: get version number"
+      echo " "
+      exit 0
+      ;;
+    -v|--verbose)
+      echo "Version: 0.1"
+      shift
```

Comment:
Hum maybe should we add a --version option just for the version and keep verbosity for other stuff ?

Reply:
Yes typo on my side